.github/workflows/rebuild-released-images.yaml

```diff
@@ -78,40 +77,54 @@ jobs:
           ref: "v${{ matrix.version }}"
           submodules: true
           fetch-depth: 0
-      - name: Set up Go
+      - name: Choose Dockerfile
+        id: pick-dockerfile
+        run: |
+          if test -f "fast.Dockerfile"; then
+            echo "dockerfile=fast.Dockerfile" >> $GITHUB_OUTPUT
+          else
+            echo "dockerfile=Dockerfile" >> $GITHUB_OUTPUT
+          fi
+      - name: Check signing supported
+        id: check-signing-support
+        run: |
+          if test -f "./scripts/sign-multiarch.sh"; then
+            echo "sign=true" >> $GITHUB_OUTPUT
+          else
+            echo "sign=false" >> $GITHUB_OUTPUT
+          fi
+      - name: Check for devbox
```

Comment:
Should we make a note of the first version using Devbox?

In fact we should probably mark all features so we can remove support as the old versions stop being supported and rebuilt.

It does not have to be in this PR though, we could make a note to tackle this separately. It is a small maintenance annotation.

Reply:
I wonder if it would be easier to just refactor this by build method:
- Version up to X supports build method 1 that means old **Dockerfile**.
- Version X-Y means build method 2 that uses **fast.Dockerfile without devbox**.
- Version Y and onward means **fast.Dockerfile with devbox**.

Still this is just food for thought and I think it is best done in a separate PR and probably separate Jira.

Reply:
We're in a rather complex situation with so many 'if' conditions in this workflow. It might be worth exploring whether those  can be delegated to their respective workflows, allowing for more automation. However, things should become simpler once we deprecate the old versions still relying on the previous Dockerfile but I do see a benefit in versioning those and being more explicing why we do that.

Reply:
yeah, but still I think that should be a different effort Jira and PR wise